src/Evolution/Systems/NewtonianEuler/Characteristics.hpp

```diff
@@ -33,9 +52,37 @@ namespace NewtonianEuler {
  * and \f$c_s\f$ is the sound speed.
  */
 template <size_t Dim>
+void characteristic_speeds(
+    gsl::not_null<std::array<DataVector, Dim + 2>*> char_speeds,
```

Comment:
And here

Reply:
Good question! No, because this is a declaration, not definition